pymc_marketing/mmm/budget_optimizer.py

```diff
@@ -409,8 +413,16 @@ def allocate_budget(
         MinimizeException
             If the optimization fails for any reason, the exception message will contain the details.
         """
+        # set total budget
         self._total_budget.set_value(np.asarray(total_budget, dtype="float64"))
 
+        # coordinate user-provided and default minimize_kwargs
+        if minimize_kwargs is None:
+            minimize_kwargs = self.DEFAULT_MINIMIZE_KWARGS.copy()
```

Comment:
Nit: no real need to copy
```suggestion
            minimize_kwargs = self.DEFAULT_MINIMIZE_KWARGS
```